### PR TITLE
feat: create cpu and mem cgroup for scope in app.slice

### PR DESCRIPTION
**Pull request by black-desk**
@zsien

看一下

### CODE REVIEW

File: system/resource_ctl/swapsched.go [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+	if err != nil {
+		logger.Infof("error: %v", err)
+	}
+	if cfg == nil || len(cfg) == 0 {
+		logger.Infof("resource_ctl disabled")
+		return nil
+	}
+
+	go func() {
+		reg, err := regexp.Compile("^/sys/fs/cgroup/systemd/(.*app-dde-(.+)-.*.scope)$")
```
Comment (zsien) [inline system/resource_ctl/swapsched.go:158]: \.scope 吧

Comment (black-desk) [reply system/resource_ctl/swapsched.go:158]: 只想处理startdde拉起来的

Comment (black-desk) [reply system/resource_ctl/swapsched.go:158]: https://github.com/linuxdeepin/startdde/blob/3ffd2934e1c20b5123b873ee9646d0a96a31e8c5/startmanager.go#L517

Comment (black-desk) [reply system/resource_ctl/swapsched.go:158]: 哦

**Pull request closed without merging**